machine/release-notes.md

```diff
@@ -7,6 +7,34 @@ redirect_from:
   - /release-notes/docker-machine/
 ---
 
+## 0.16.2 (2019-09-02)
+
+### General
+
+* Compile with Golang 1.12.9.
+* Fix VTX detection.
+
+### Drivers
+
+`amazonec2`
+* Add ssh-port flag
+* Add eu-north-1 zone
+
+`vmwarefusion`
+* Add nonempty flag to fix share-folder bug
+
+`hyperv`
+* Allow localized names for Virtual Switch
+
+`google`
+* Add support for non-default service account
+
+`openstack`
+* Add flag for metadata
+
+`softlayer`
+* Don't set the request method again
+
 ## 0.16.0 (2018-11-08)
```

Comment:
Looks like 0.16.1 is missing as well, could you add that as well? https://github.com/docker/machine/releases/tag/v0.16.1

Reply:
Thank you @thaJeztah. Will merge this PR and create a new PR for 0.16.1